trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-link-to-premium-payments-tab.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getLinkToPremiumPaymentsTab } from './get-link-to-premium-payments-tab';
+
+console.error = jest.fn();
+
+describe('getLinkToPremiumPaymentsTab', () => {
+  const reportId = '123';
+
+  it('returns URL without params when selectedFeeRecordIds array is empty', () => {
+    // Arrange
+    const selectedFeeRecordIds: number[] = [];
+
+    // Act
+    const result = getLinkToPremiumPaymentsTab(reportId, selectedFeeRecordIds);
+
+    // Assert
+    expect(result).toBe('/utilisation-reports/123');
```

Comment:
Can we use the `reportId` const in the assertions?

Reply:
This is perhaps simple enough that it doesn't matter too much, but actually part of the test is testing that we do the string interpolation correctly, so if we use string interpolation in the assertion we are partly reimplementing the function under test.

Reply:
Using the `reportId` in the assertion is cleaner and more DRY - it's currently hard coded and repeated.

It doesn't make sense to call the `getLinkToPremiumPaymentsTab` function with the mock ID and then assert without the ID.

I don't think it's partly reimplementing the function. The function simply consumes this param so whatever this param is, will be in the result.

Additionally, with the current approach, if for some reason the value of `reportId` is changed, other instances would need to be manually updated; This is avoided if we re-use the mock ID